var/spack/repos/builtin/packages/musl/package.py

```diff
@@ -6,7 +6,7 @@
 from spack import *
 
 
-class Musl(Package):
+class Musl(MakefilePackage):
```

Comment:
I would still keep this as a `Package` base class since it doesn't come with a Makefile. This definitely feels like a custom build system.